lib/basedriver/driver.js

```diff
@@ -259,7 +259,7 @@ class BaseDriver extends Protocol {
     this.clearNewCommandTimeout();
 
     // if we don't have this command, it must not be implemented
-    if (!this[cmd]) {
+    if (!this[cmd] && !getImgElFromArgs(args)) {
```

Comment:
you could probably set the result of `getImgElFromArgs(args)` to a separate const variable, since it is called twice